container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/enclave_object_attributes_provider.go

```diff
@@ -188,6 +188,12 @@ func (provider *kubernetesEnclaveObjectAttributesProviderImpl) ForUserServicePod
 	}
 	labels[kubernetes_label_key.LogsServiceUUIDKubernetesLabelKey] = uuidLabelValue
 
+	nameLabelValue, err := kubernetes_label_value.CreateNewKubernetesLabelValue(string(serviceName))
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "An error occurred creating a short GUID Kubernetes label value from GUID string '%v'", string(serviceName))
```

Comment:
Careful with the copy-pasted log messages; this looks like it wasn't modified from the line above